src/scripts/main.js

```diff
@@ -1,3 +1,178 @@
 'use strict';
 
-// write code here
+const tableHeaders = document.querySelectorAll('th');
+const rowsArray = Array.from(document.querySelectorAll('tbody tr'));
+const sortDirection = {};
+
+function formatSalary(cellValue) {
+  return parseFloat(cellValue.replace(/[$,]/g, ''));
+}
+
+function sortRows(index, isAscending, rowsArr) {
```

Comment:
The function `sortRows` is doing too much. It's sorting the rows and also formatting the salary. It would be better to separate these responsibilities into different functions.